modules/cnv-preparing-nodes-to-enable-dedicated-resources.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv_advanced_vm_management/cnv-dedicated-resources-vm.adoc
+// * cnv/cnv_vm_templates/cnv-dedicated-resources-vm-template.adoc
+
+[id="cnv-preparing-nodes-to-enable-dedicated-resources_{context}"]
+
+= Preparing to enable dedicated resources
+
+To prepare to enable dedicated resources, the `cpumanager` label must be `true`
+for each node on which you want to schedule virtual machines with
+dedicated resources.
+
+If the `cpumanager` label isn't applied to an available node,
+the workload can not be scheduled.
+
+.Procedure
+Ensure that the `cpumanager` label is `true` for each node on which you want
+to schedule virtual machines with dedicated resources.
+
+. Verify that CPU Manager is running on the CPU node whose resources you want to dedicate.
```

Comment:
If we keep this module, it should tell users how to verify that CPU manager is set up, (which I think might be:
`$ oc get machineconfigpool worker -o yaml` ?)
And how to verify that the `cpumanager = true` label is applied (which might be:
`$ oc get node <node_name> -o yaml | grep 'cpumanager:'` ?)

Reply:
Spoke with Tomas and he says that if the `cpumanager = "true"` label is applied to a node, then that is good enough. 
If a user doesn't have cpumanager configured properly and adds that label themselves, CNV will change the label back to false.

Reply:
I'm removing this module from both assemblies.